CMakeLists.txt

```diff
@@ -92,11 +92,14 @@ if (NOT CPACK_GENERATOR)
   elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "ubuntu" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "debian")
     set(CPACK_GENERATOR DEB CACHE STRING "CPack generator to use, e.g. {DEB, RPM, TGZ}." FORCE)
     message(STATUS "Setting unspecified CPACK_GENERATOR to ${CPACK_GENERATOR}. This is the correct setting for normal builds.")
-  elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "fedora" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "rocky")
-    set(CPACK_GENERATOR RPM CACHE STRING "CPack generator to use, e.g. {DEB, RPM, TGZ}." FORCE)
-    message(STATUS "Setting unspecified CPACK_GENERATOR to ${CPACK_GENERATOR}. This is the correct setting for normal builds.")
   else()
-    message(FATAL_ERROR "Failed to set CPACK_GENERATOR for unknown Linux Distribution: ${MUNGEFS_LINUX_DISTRIBUTION_NAME}")
+    set(MUNGEFS_SUPPORTED_LINUX_DISTRIBUTION_NAMES "centos" "opensuse" "fedora" "almalinux" "rocky")
```

Comment:
add RPM to the name...?

Reply:
which one do you like?
- MUNGEFS_SUPPORTED_LINUX_DISTRIBUTION_NAMES_RPM
- MUNGEFS_SUPPORTED_RPM_LINUX_DISTRIBUTION_NAMES
- MUNGEFS_SUPPORTED_RPM_BASED_LINUX_DISTRIBUTION_NAMES
- MUNGEFS_RPM_BASED_LINUX_DISTRIBUTION_NAMES

Reply:
going with this one: `MUNGEFS_SUPPORTED_RPM_BASED_LINUX_DISTRIBUTION_NAMES`